src/cmakeTools.ts

```diff
@@ -2333,11 +2315,13 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
         await this.workspaceContext.state.reset();
     }
 
-    private _sourceDir = '';
+    // Don't get this from the driver. Source dir is required to evaluate presets.
+    // Presets contain generator info. Generator info is required for server api.
+    private srcDir = '';
```

Comment:
Because we only have a `get` here, it's ok to leave this one unchanged as `_sourceDir`